tasks/main.yml

```diff
@@ -54,13 +54,14 @@
     backup: true
   become: true
 
-- name: Generate new grub-config
+- name: Generate new grub-config (RedHat)
   command: grub2-mkconfig
   register: grub_mkconfig_result
   become: true
   changed_when: false
+  when: ansible_os_family == "RedHat"
```

Comment:
Could we go with `ansible_facts.os_family` here and elsewhere?

Reply:
If only there was a blog post that articulated the benefit of doing so!